src/plugins/workspace/public/components/workspace_form/use_workspace_form.ts

```diff
@@ -107,34 +107,34 @@ export const useWorkspaceForm = ({
         return;
       }
 
-      onSubmit?.(
-        {
-          name: currentFormData.name!,
-          description: currentFormData.description,
-          color: currentFormData.color || '#FFFFFF',
-          features: currentFormData.features,
-          permissionSettings: currentFormData.permissionSettings as WorkspacePermissionSetting[],
-          selectedDataSourceConnections: currentFormData.selectedDataSourceConnections,
-        },
-        true
-      );
+      const submitFormData = {
+        name: currentFormData.name!,
+        description: currentFormData.description,
+        color: currentFormData.color || '#FFFFFF',
+        features: currentFormData.features,
+        permissionSettings: currentFormData.permissionSettings as WorkspacePermissionSetting[],
+        selectedDataSourceConnections: currentFormData.selectedDataSourceConnections,
+      };
```

Comment:
Nit: we can have a unified function like `getSubmitFormData` to generate the data payload for submit, it can be used here and line 130.

Reply:
Thanks, the code has been updated